# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/image-element/reftest.list
# Commit: fc2b3d6448bc
# Full Hash: fc2b3d6448bcc3575fffb1abc923e7e703aeef54
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 2 - Performance and quality fixes for part 1. r=gfx-reviewers,nical
#   
#   * Add support for local scale factors to a surface, allowing it to
#     be rasterized in root coordinate space. This allows snapping to
#     work across surfaces where the surface transform is a fractional
# ==============================================================================

diff -r 8c70bc92ad36 -r fc2b3d6448bc layout/reftests/image-element/reftest.list
--- a/layout/reftests/image-element/reftest.list	Wed Feb 23 20:49:26 2022 +0000
+++ b/layout/reftests/image-element/reftest.list	Wed Feb 23 20:49:27 2022 +0000
@@ -16,7 +16,7 @@
 fuzzy-if(d2d&&/^Windows\x20NT\x206\.1/.test(http.oscpu),0-16,0-90) == element-paint-background-size-01.html element-paint-background-size-01-ref.html
 == element-paint-background-size-02.html element-paint-background-size-02-ref.html
 fuzzy(0-255,0-4) random-if(/^Windows\x20NT\x206\.1/.test(http.oscpu)) == element-paint-transform-repeated.html element-paint-transform-repeated-ref.html # Bug 1475907
-fuzzy-if(d2d,0-255,0-24) fuzzy-if(!useDrawSnapshot,255-255,56-72) == element-paint-transform-03.html element-paint-transform-03-ref.html
+fuzzy-if(d2d,0-255,0-24) fuzzy-if(!useDrawSnapshot,255-255,80-115) == element-paint-transform-03.html element-paint-transform-03-ref.html
 
 # For !nativeThemePref: element() uses fallback / skia in WebRender, which antialiases differently from WR.
 # For !nativeThemePref+Windows+WebRender: bug 1496542, the scrollframe snaps differently.